### ISSUE TITLE
Cannot install 0.9.9 through an updatesite

### ISSUE THREAD

**Issue by markleusink** [klehmann/domino-jna#12]
Just downloaded the 0.9.9 version from [this](https://github.com/klehmann/domino-jna/releases/download/0.9.9/domino-jna-xpages-0.9.9.zip) link. Extracted the downloaded ZIP and imported the plugin in my updatesite.nsf database. Restarted the http task, but the plugin doesn't load: a `tell http osgi ss com.mindoo` on the server returns a blank list. Any ideas?

Tested on CentOS 6 64, Domino 901 FP9.

**Comment by klehmann** [comment]
Weird. I just updated my Win64 dev server from FP8 to FP9 and the plugin is still being loaded and working.
I have another Domino server with 9.0.1 FP8 on Linux and the plugin is working as well. Will need to update that one to check if it's a Linux/FP9 specific issue.

The only idea I have is
1) make sure OSGI_DYNAMIC_BUNDLES contains the update site database
2) sign all documents in the update site db with an Admin ID (Actions / Sign All Content) followed by a "restart task http"
3) try to add another plugin to the update site db to see if it's really a Domino JNA issue

**Comment by markleusink** [comment]
I installed 0.9.9 on a Win64 dev server too and it installed ok.

Rechecked the CentOS server, but it didn't install.

The updatesite database I was using had some older version of JNA in it, as well as older versions of other plugins. So I tried something else: create a new updatesite database and install the latest versions of all plugins there. Updated the notes.ini and restarted the server. JNA installs fine now. Still weird, but case closed 👍
